packages/components/molecules/f-skeleton-loader/src/components/skeletons/Sentence.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div>
-        <text-block :class="$style['c-sentence-textBlock']" />
-        <text-block :class="$style['c-sentence-textBlock']" />
+        <text-block :class="$style['c-skeletonLoader-text']" />
```

Comment:
Just to confirm – this one looks correct because it's not named specifically to be tied to restaurantCard, so it makes sense to be named generically like this.